Cargo.toml

```diff
@@ -1,14 +1,15 @@
 [workspace]
 resolver = "2"
 members = [
-    "base",
-    "client",
-    "client/feature",
-    "errors",
-    "marshal",
-    "marshal/derive",
-    "service",
-    "unionify"
```

Comment:
Prefixing every directory with `tpm2_rs` feel unnecessary. I strongly prefer prefix-free directories. This way we won't won't have to `cd tpm2-rs-` every time we want to go to somewhere. It makes paths longer than they need to be, especially that we are in tpm-rs repository anyways.

Reply:
I find the clarity of using the actual crate name to be a nice property. If other directories end up in our root like `docs` or `norms` or something like that, it'll be clear what's what, and it keeps them organized closely together.

There's one other reason I think we might want to prefix these directories: A future exists where we make the base crate suffix-less (see: https://github.com/tpm-rs/tpm-rs/issues/65).

I think in that case the list of crates in this directory will be:

*   tpm2
*   tpm2-client
*   tpm2-client-features
*   tpm2-errors
*   tpm2-marshal
*   tpm2-marshal-derive
*   tpm2-service
*   tpm2-unionify
*   tpm2-unionify-derive

Reply:
If you want, we can do prefix-free for now but revisit this after https://github.com/tpm-rs/tpm-rs/issues/65. I still prefer having the prefix, but if you really don't like it I won't block on it.

Reply:
I personally don't prefer it. Think about it, it adds no extra information, even docs is technically `tpm2-rs-docs`. So what `tpm2-rs` would signify that this is code while other parts are not code. However I don't think this distinction is really necessary at this point, and people can just `cd` in and see what is in there.

Reply:
docs wouldn't be `tpm2-rs-docs`, it would have just been `docs`, like this:

* docs  // Because this is not a crate
* tpm2-rs-base
* tpm2-rs-client
* tpm2-rs-client-features
* tpm2-rs-... etc. etc.

I disagree, but not strongly enough to push back about it. I'll update the PR.